sp/src/game/server/hl2/npc_citizen17.cpp

```diff
@@ -145,6 +145,7 @@ ConVar npc_citizen_gib( "npc_citizen_gib", "1" );
 ConVar npc_citizen_longfall_death_height("npc_citizen_longfall_death_height", "32768");
 ConVar npc_citizen_longfall_glow_chest_jump_fade_enter( "npc_citizen_longfall_glow_chest_jump_fade_enter", "0.25" );
 ConVar npc_citizen_longfall_glow_chest_jump_fade_exit( "npc_citizen_longfall_glow_chest_jump_fade_exit", "0.5" );
+ConVar npc_citizen_longfall_test_high_ground( "npc_citizen_longfall_test_high_ground", "0", FCVAR_NONE, "Should longfall / jump citizens only try to shoot from positions above their target?" );
```

Comment:
This is disabled by default. Is that intentional?

Reply:
Yes! I figured it should be a convar so we can toggle it off if necessary